docs/source/installing.rst

```diff
@@ -61,6 +61,10 @@ To start the installation process using the Boards Managaer, follow these steps:
 Windows
 -------
 
+.. warning:: Arduino ESP32 core v2.x.x cannot be used on Windows 7 or earlier. The Windows 32 bits OS is no longer supported by this toolchain.
```

Comment:
Windows 8.x should be listed here as well, there was another issue for Windows 8.1 x86 (32bit)

Reply:
Thanks!